client/tests/TransportSpec.elm

```diff
@@ -0,0 +1,27 @@
+module TransportSpec exposing (..)
+
+import Test exposing (..)
+import Test.Html.Query as Query
```

Comment:
Oo, I didn't know about that package. Looks like a good starting point as far as unit tests are concerned. Looking forward to seeing what comes next. :)

Reply:
I'm trying to hack my way into behaviour-driven elm, which ATM seems to be a rather lonely path. Check this out as well: https://github.com/brian-watkins/elmer